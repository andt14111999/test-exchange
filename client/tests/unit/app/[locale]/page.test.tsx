import React from "react";
import { render } from "@testing-library/react";
import HomePage from "@/app/[locale]/page";

// Mock HomeContent
jest.mock("@/components/home-content", () => ({
  HomeContent: () => <div data-testid="home-content" />,
}));

describe("HomePage", () => {
  it("should render HomeContent", () => {
    const { getByTestId, container } = render(<HomePage />);
    expect(getByTestId("home-content")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
