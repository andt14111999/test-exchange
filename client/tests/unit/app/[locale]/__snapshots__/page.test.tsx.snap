// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render HomeContent 1`] = `
<div>
  <div
    data-testid="home-content"
  />
</div>
`;
